### PR TITLE
Zip / deflate support for AIP compression

### PR DESCRIPTION
**Pull request by helenst**
Provide zip/deflate capability in the AIP compression option

This connects to https://github.com/archivematica/Issues/issues/493.

### CODE REVIEW

File: src/MCPClient/lib/clientScripts/compress_aip.py [hunk 1]
```diff
@@ -69,6 +69,17 @@ def compress_aip(job, compression, compression_level, sip_directory, sip_name, s
             'echo program="7z"\; '
             'algorithm="{}"\; '
             'version="`7z | grep Version`"'.format(compression_algorithm))
+    elif program == 'zip':
+        compressed_location = uncompressed_location + ".zip"
+        command = '/usr/bin/7z a -bd -tzip -y -mm={algorithm} -mx={level} -mtc=on -mmt=on "{compressed_location}" "{uncompressed_location}"'.format(
```
Comment (alexwlchan) [inline src/MCPClient/lib/clientScripts/compress_aip.py:97]: At a quick glance, it looks like `uncompressed_location` is built from the user-provided name of the SIP, which mean something suitably annoying could escape this quoting.

Maybe `shlex.quote()` could be a safer way to escape those filenames?

(But I acknowledge that breaks the convention established in the rest of the file, so feel free to ignore me!)

File: src/MCPClient-base.Dockerfile [hunk 2]
```diff
@@ -62,6 +62,7 @@ RUN set -ex \
 		ufraw \
 		unrar-free \
 		uuid \
+		unzip \
```
Comment (sevein) [inline src/MCPClient-base.Dockerfile:63]: Is this needed by 7z?

Comment (helenst) [reply src/MCPClient-base.Dockerfile:63]: It's needed by `atool` when it tries to decompress during AIP verification.

Comment (sevein) [reply src/MCPClient-base.Dockerfile:63]: Oh I had no idea, thanks!

**Comment by helenst** [comment]
Closing this since the related issue was closed.

**Pull request closed without merging**